src/upnp/headers.cc

```diff
@@ -82,7 +82,7 @@ std::pair<std::string, std::string> Headers::parseHeader(const std::string& head
     return { first, second };
 }
 
-void Headers::writeHeaders(UpnpFileInfo* fileInfo) const
+void Headers::writeHeaders(const UpnpFileInfo* fileInfo) const
 {
 #if defined(USING_NPUPNP)
     std::copy(headers.begin(), headers.end(), std::back_inserter(fileInfo->response_headers));
```

Comment:
The change breaks NPUPNP build

Reply:
Yeah, cppcheck only warned on the read function. I was overzealous. Will fix.

Reply:
Fixed.